.gitignore

```diff
@@ -15,3 +15,4 @@
 /log/*
 !/log/.keep
 /tmp
+/vendor/cache/*
```

Comment:
We're currently caching gems. But I don't know that we need to be any more as our internet connection here seems solid...